meeteval/io/__main__.py

```diff
@@ -0,0 +1,132 @@
+import os
+import sys
+from stat import S_ISFIFO
+from pathlib import Path
+import argparse
+import meeteval
+
+# formats = {
+#     'ctm': meeteval.io.ctm.CTM,
+#     'stm': meeteval.io.stm.STM,
+#     'seglst': meeteval.io.seglst.SegLST,
+#     'rttm': meeteval.io.rttm.RTTM,
+# }
+
+# d = meeteval.io.load('').to_seglst()
+# for s in d:
+#     s['words'] = '<NA>'
+# meeteval.io.STM.new(d).dump('.stm')
+
+
+def is_piping_in():
+    """
+    Checks whether input is being piped into the command.
+    """
+    return S_ISFIFO(os.fstat(0).st_mode)
+
+
+def convert(input_files, output_file, input_format, output_format, **kwargs):
+    data = []
+
+    for f in input_files:
+        d = meeteval.io.load(f, format=input_format).to_seglst()
+        
+        extra = {}
+
+        if isinstance(f, (str, Path)):
```

Comment:
Check that `stat.S_ISREG(os.stat(f).st_mode)` is True.
```
$ python -c "import sys,os,stat; print(sys.argv[1]); [print(f, getattr(stat, f)(os.stat(sys.argv[1]).st_mode)) for f in ['S_ISREG', 'S_ISFIFO', 'S_ISCHR', 'S_ISBLK']]" <(echo "test")
/dev/fd/63
S_ISREG False
S_ISFIFO True
S_ISCHR False
S_ISBLK False
```

Reply:
Done